### DISCUSSION TITLE
Upgrading to 1.8.7 - Initializing Service to a method deprecated in 2.0

### DISCUSSION THREAD

**Discussion by KroaX83** [hangfire-postgres/Hangfire.PostgreSql#341] [Q&A] [answered]
Hi,

I've recently updated to the latest Hangfire version. In Program.cs it shows that my service initialization is using a method that will be deprecated in version 2.0.

![image](https://github.com/hangfire-postgres/Hangfire.PostgreSql/assets/61991098/e246d0c4-e6f8-4ee9-82ec-bde3fe7821c0)

Implementation note:
![image](https://github.com/hangfire-postgres/Hangfire.PostgreSql/assets/61991098/cc8a5513-cbdd-4df9-9735-5388d271a591)

I'm struggling to find how this service initialization should change to use the correct overload once 2.0 rolls around, any help will be greatly appreciated. Thank you.

**Comment by azygis** [comment, marked as answer]
It's actually in the readme:

```
services.AddHangfire(config =>
    config.UsePostgreSqlStorage(c =>
        c.UseNpgsqlConnection(Configuration.GetConnectionString("HangfireConnection"))));
```

**Reply by KroaX83** [reply]
Thank you for the help, got it working again.
